test-suite/Makefile

```diff
@@ -252,7 +252,12 @@ $(addsuffix .log,$(wildcard bugs/closed/*.v)): %.v.log: %.v
 	@echo "TEST      $< $(call get_coq_prog_args_in_parens,"$<")"
 	$(HIDE){ \
 	  echo $(call log_intro,$<); \
-	  $(coqc) "$<" $(call get_coq_prog_args,"$<") 2>&1; R=$$?; times; \
+	  (echo "\
+	     bugs/closed/bug_3783.v \
+	     bugs/closed/bug_4157.v \
+	     bugs/closed/bug_5127.v \
```

Comment:
@proux01 did you manage to understand why these files fail? (anyway I am not suggesting to change these lines: I was just wondering if a related issue should be opened)

Reply:
I guess, they generate term that are too large for native compute, but this could be investigated further.